mitosheet/mitosheet/utils.py

```diff
@@ -156,17 +156,9 @@ def df_to_json_dumpsable(
     """
 
     (num_rows, num_columns) = original_df.shape 
+    df = original_df.iloc[: , :max_columns]
```

Comment:
I was confused why we needed to do this line here, since we're also doing it in the `convert_df_to_parsed_json` function, and I think I found an existing bug (I think one that I introduced). 

I think we should update line line 192 to use the `original_df` instead of `df` because we want all of the ColumnIDs, not just the first max_columns to be part of the map. 

And then we could also update the construction of `column_final_data` and `column_dtype_map` to use `df` instead of `original_df`. But this doesn't matter because we're looping over the results of `json_obj`.